evalml/tests/component_tests/test_time_series_baseline_estimators.py

```diff
@@ -1,58 +0,0 @@
-import numpy as np
```

Comment:
Logic that used to be tested here is tested in `test_time_series_baseline_pipeline` .